src/components/FormField.js

```diff
@@ -0,0 +1,121 @@
+const React = require('react');
+const Radium = require('radium');
+
+const StyleConstants = require('../constants/Style');
+
+class FormField extends React.Component {
+  render () {
+    return (
+      <div
+        className='mx-form-field-component'
+        style={[
+          styles.component, this.props.style,
+          { height: this.props.height + 'px', width: this.props.width + 'px' }
+        ]}
+      >
+        <div
+          className='mx-form-field-label'
+          style={[
+            styles.label, this.props.labelStyle,
+            { marginTop: this.props.height - 16 }
```

Comment:
Trying to understand this line here, so I set the height of the component to 300px and then the label would have a top margin of 284px?